R/extract_param.R

```diff
@@ -37,7 +40,7 @@
 #' # extract parameters of a gamma distribution from median and range
 #' extract_param(
 #'   type = "range",
-#'   values = c(8, 6, 13),
+#'   values = c(10, 3, 18),
 #'   distribution = "gamma",
 #'   samples = 20
 #' )
```

Comment:
How is the extraction step now interacting with the database? I remember you previously mentioned it would sit somewhere between the raw database and the `epiparam` object (rather than having extracted parameters in database as before)? Or have I misunderstood?

Reply:
The database contains only values reported in the primary literature. If summary statistics are provided but distribution parameter estimates are not, these are not added at the database compilation stage. The database should be an unaltered compilation of the literature, which is different from the previous version of the database. In the new version, if parameter estimates are not provided but adequate summary statistics are then `calc_dist_param()` will work out the best way to estimate those estimates and append them to the `epidist` object. This calculation happens automatically (by default but can be turned off) when making an `epidist` object from an `epiparam` object. The `epiparam` object when read in from the package is unchanged and so does not include parameter estimates, this could be easily changes if required.